### PR TITLE
datcyx new card

### PR DESCRIPTION
**Pull request by Datcyx**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2812,6 +2812,32 @@ <h2>A project for first-time contributions</h2>
           </div>
         </div>
         <!-- ________ *Rabblerag* Contributor card END ________  -->
+		
+		  <!-- ________ *DATCYX* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Dionisio Datiles III</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/" target="#">None</a>
```
Comment (TenTraicion) [inline index.html:2821]: Replace username with your twitter username. if you don't have twitter replace twitter in this line and the line before with github and, replace the username with your github username. 
```suggestion
            <a href="https://www.twitter.com/username" target="_blank">@username</a>

```

File: index.html [hunk 2]
```diff
@@ -2812,6 +2812,32 @@ <h2>A project for first-time contributions</h2>
...
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#" target="https://www.w3schools.com/" title="W3schools">Web Development for beginner(W3schools)</a>
+              </li>
+              <li>
+                <a href="#" target="https://education.github.com/pack/offers" title="Github Student">Have Your Cloud Server by Applying on Github Student Pack</a>
+              </li>
+              <li>
+                <a href="#" target="https://itsourcecode.com/" title="IT source code">ITsourceCode(A totorial website of my IT Teacher)</a>
```
Comment (TenTraicion) [inline index.html:2835]:
```suggestion
                <a href="https://www.w3schools.com/" target="_blank" title="W3schools">Web Development for beginner(W3schools)</a>
              </li>
              <li>
                <a href="https://education.github.com/pack/offers" target="_blank" title="Github Student">Have Your Cloud Server by Applying on Github Student Pack</a>
              </li>
              <li>
                <a href="https://itsourcecode.com/" target="_blank" title="IT source code">ITsourceCode(A totorial website of my IT Teacher)</a>

```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@Datcyx , kindly make the requested changes and commit. Thanks!

**Comment by Datcyx** [comment]
I'm sorry, I don't notice my wrong, I have a bad day when I did this

**Comment by TenTraicion** [comment]
> I'm sorry, I don't notice my wrong, I have a bad day when I did this

@Datcyx, No sweat, but you still need to fix your twitter handel.

**Comment by Datcyx** [comment]
I don't have twitgee

**Comment by Datcyx** [comment]
Hi I updated my twitter now

**Review by TenTraicion — APPROVED** [review]

**Comment by TenTraicion** [comment]
@Datcyx, Congratulation on your PR Merge, and Thanks for your Contribution. You can check your card in our [Live Website](https://syknapse.github.io/Contribute-To-This-Project/)! It may take 2-3 minutes to make your card visible in our website. 

Welcome to Open Source Community. You can follow the [Next Steps](https://github.com/Syknapse/Contribute-To-This-Project#next-steps) now. You are also welcome to join our [Discord Server: https://discord.gg/tWkvS4ueVF](https://discord.gg/tWkvS4ueVF) and be a part of our community.

**Pull request merged**